org.kde.AwesomeBump.json

```diff
@@ -0,0 +1,53 @@
+{
+    "app-id": "org.kde.AwesomeBump",
+    "branch": "Release",
+    "build-options": {
+        "no-debuginfo": true,
```

Comment:
why do you need this? debuginfo should be stripped and put into an optional .Debug extension automatically.

Reply:
Thanks, I don't need this. It will be removed in the next commit.